WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/ShippingLabelFormViewModel.swift

```diff
@@ -29,6 +30,25 @@ final class ShippingLabelFormViewModel {
 
 // MARK: - Utils
 extension ShippingLabelFormViewModel {
+    // We generate the default origin address using the information
+    // of the logged Account and of the website.
+    private static func getDefaultOriginAddress() -> ShippingLabelAddress? {
+        // TODO: 2973 - for populating the origin address with the correct first and last name of the store owner
+        // we need to explicitly ask for first_name,last_name parameters in loadAccountSettings method
+        // and add the new properties to AccountSettings entity.
+        let address = Address(firstName: ServiceLocator.stores.sessionManager.defaultAccount?.displayName ?? "",
+                lastName: "", company: ServiceLocator.stores.sessionManager.defaultSite?.name ?? "",
+                address1: SiteAddress().address,
+                address2: SiteAddress().address2,
+                city: SiteAddress().city,
+                state: SiteAddress().state,
+                postcode: SiteAddress().postalCode,
+                country: SiteAddress().countryName ?? "",
+                phone: "",
```

Comment:
Is the phone something we can also retrieve from the back end, or does this always have to be entered manually?

Reply:
For the moment the source address will not contain the phone number, because it's not information we have on the API side.